src/Sylius/Behat/Resources/config/services/pages/admin/catalog_promotion.xml

```diff
@@ -17,7 +17,7 @@
 >
     <parameters>
         <parameter key="sylius.behat.page.admin.catalog_promotion.create.class">Sylius\Behat\Page\Admin\CatalogPromotion\CreatePage</parameter>
-        <parameter key="sylius.behat.page.admin.catalog_promotion.index.class">%sylius.behat.page.admin.crud.index.class%</parameter>
+        <parameter key="sylius.behat.page.admin.catalog_promotion.index.class">Sylius\Behat\Page\Admin\CatalogPromotion\IndexPage</parameter>
```

Comment:
Why? Somebody might be using the old behavior somehow.

Reply:
The old behaviour is that there is no specific class for catalog promotion index, so if a developer needs to add things there, he has already extends the index page himself/herself.

Moreover, Behat is not covered by your breaking change policy.